pkg/deploy/util/util.go

```diff
@@ -233,6 +235,66 @@ func MakeDeployment(config *deployapi.DeploymentConfig, codec runtime.Codec) (*a
 	return deployment, nil
 }
 
+// GetReplicaCountForDeployments returns the sum of all replicas for the
+// given deployments.
+func GetReplicaCountForDeployments(deployments []api.ReplicationController) int {
+	totalReplicaCount := 0
+	for _, deployment := range deployments {
+		totalReplicaCount += deployment.Spec.Replicas
+	}
+	return totalReplicaCount
+}
+
+// GetActualReplicaCountForDeployments returns the sum of actual replicas for the
+// given deployments.
+func GetActualReplicaCountForDeployments(deployments []api.ReplicationController) int {
```

Comment:
"Actual" isn't making a meaningful distinction for me here - the above method returns the desired replicas, so perhaps "Actual" would mean more if that method was `GetDesiredReplicaCountForDeployments` our similar?